### ISSUE TITLE
GetModuleFileNameA signature seems to be incorrect

### ISSUE THREAD

**Issue by coffeecup-winner** [Vector35/binaryninja-api#2378]
The signature I see seems to match [GetModuleFileNameExA](https://docs.microsoft.com/en-us/windows/desktop/api/psapi/nf-psapi-getmodulefilenameexa), not [GetModuleFileNameA](https://docs.microsoft.com/en-us/windows/win32/api/libloaderapi/nf-libloaderapi-getmodulefilenamea).

![image](https://user-images.githubusercontent.com/774461/115126737-4c54ec00-9f86-11eb-89c3-3f7c063eb2d6.png)

In the decompiled IL code the suggested arguments don't make sense, but would if the correct signature was assumed instead.

**Comment by xusheng6** [comment]
This is fixed before `3.2.3946-dev (Build ID a5936bc8)`

```
>>> bv.import_library_object('GetModuleFileNameA')
<type: immutable:FunctionTypeClass 'uint32_t __stdcall(HINSTANCE hModule, uint8_t* lpFilename, uint32_t nSize)'>
```

**Comment by coffeecup-winner** [comment]
Thanks!
